docs/man/keylime_registrar.8.rst

```diff
@@ -0,0 +1,109 @@
+=====================
+keylime_registrar.8
+=====================
+
+------------------------------
+Keylime registrar service
+------------------------------
+
+:Manual section: 8
+:Author: Keylime Developers
+:Date: September 2025
+
+NAME
+====
+
+keylime_registrar - Keylime registrar service for agent registration
+
+SYNOPSIS
+========
+
+**keylime_registrar**
+
+(Most operations require root privileges, use with sudo)
+
+DESCRIPTION
+===========
+
+The registrar is a long-running HTTP(S) service used by agents and the verifier during
+registration and activation. The service does not accept command-line options; behavior is
+configured via configuration files and environment variables and managed by keylime tenant.
```

Comment:
It seems there are too many "and"s here:
```configured via configuration files and environment variables and managed by keylime tenant.```
It might be better something like:
```configured via configuration files and environment variables, and is managed by keylime tenant.```